### DISCUSSION TITLE
play timeline widget in a loop

### DISCUSSION THREAD

**Discussion by Vbsasuke26** [orbat-mapper/orbat-mapper#216] [Ideas]
Hi Kjell. Will it be possible to play timeline in a loop by providing start time and end time so we can see units moving automatically ?

**Comment by kjellmf** [comment]
Yes, I plan to add simple playback features. I have not figured out the internal data-structures and user interface for it yet. I've added #217 to track this feature request.

**Comment by kjellmf** [comment]
This functionality has now been implemented. It is still experimental and currently only available through the map context menu or command palette.
![image](https://github.com/orbat-mapper/orbat-mapper/assets/29217/42074486-d359-4c7d-b4c0-ce1371fd6329)

**Reply by Vbsasuke26** [reply]
Wow. This is so so great. I saw u have marked tasks completed but I didn't know how to access this feature. I tried  main context menu but I should have seen GitHub notification. Thank you so much for this addition. It will be really helpful. 🥇

**Reply by kjellmf** [reply]
I will find a better place for the playback menu. It should probably be available in the main menu. My initial plan was to put it in the map toolbar, but it is a bit crowded there.

**Comment by kjellmf** [comment]
This feature has now been implemented. Playback controls are available in the main navigation bar.
![image](https://github.com/orbat-mapper/orbat-mapper/assets/29217/b4ea0b16-254d-4fbc-92c0-cefca7ffc55c)
